### ISSUE TITLE
Accessibility - cannot navigate through form panes (tabs) via the keyboard.

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/hungry-hungry-hippo#1060]
Notice how tabbing through the form does not reach the left navigation, only the active pane. The left nave must be tab navigable in order to be accessible.

https://github.com/user-attachments/assets/e0102610-d83a-43d6-90ca-6b7e0e080505

**Comment by justinlittman** [comment]
Verified this is still a problem.
